waku/factory/external_config.nim

```diff
@@ -76,6 +76,12 @@ type WakuNodeConf* = object
     name: "rln-relay-eth-contract-address"
   .}: string
 
+  rlnRelayChainId* {.
+    desc: "Chain ID of the provided contract (optional, will fetch from RPC provider if not used)",
+    defaultValue: 0,
```

Comment:
So if not provided it will default to `0` but technically that can be a valid value?

Reply:
0 is an invalid value for chainId. avoided using an `Optio n` here because of that